streams/src/main/java/org/apache/kafka/streams/StreamsBuilder.java

```diff
@@ -567,7 +392,7 @@
     public synchronized <K, V> KTable<K, V> table(final TimestampExtractor timestampExtractor,
```

Comment:
This overload can be removed?

Reply:
No it can't be yet, but will be in the future once the `Materialized` class (which is part of another pending PR) is in. I can't put the `Materialized` interface in until  https://github.com/apache/kafka/pull/3767 is merged

Reply:
Ack.